### DISCUSSION TITLE
[BUG] [SP-API] [Issue] Keep getting Error Code: MD1000 when trying to add new store on my developer application

### DISCUSSION THREAD

**Discussion by aelsaidy** [amzn/selling-partner-api-models#3081] [General]
Hi,
Has anyone had this issue recently?

![image](https://user-images.githubusercontent.com/18082351/198354380-b526539f-8ace-4683-ade3-8159c18c7040.png)

the same authorization URL that looks like : 
https://sellercentral.amazon.com/apps/authorize/consent?<<application ID>>&version=beta

keep failing with me , in 2 different developer account profiles, 

after it was working just fine morning and then not working at the end of the day 
for 2 days !!

**Comment by sungolivia** [comment]
hey @aelsaidy  check out the example auth url from below blog post!
https://developer-docs.amazon.com/sp-api-blog/docs/troubleshooting-selling-partner-api-authorization-errors
